.gitlab/scripts.yml

```diff
@@ -169,7 +169,7 @@
     - tar -xzf $DEV_PKG_NAME.tar.gz
     - cd $DEV_PKG_NAME
 
-    - env INSTALL_DIR=$INSTALL_DIR SPEC=$SPEC SPACK_PKG_NAME=$SPACK_PKG_NAME BUILD_ALLOC="" SCRIPT_DIR=$SCRIPT_DIR
+    - env INSTALL_DIR=$INSTALL_DIR DEV_PKG_SPEC=$DEV_PKG_SPEC SPACK_PKG_NAME=$SPACK_PKG_NAME BUILD_ALLOC="" SCRIPT_DIR=$SCRIPT_DIR
```

Comment:
The DEV_PKG_NAME env variable previously was the name of the final tar file we stored, it looks like we're still extracting DEV_PKG_NAME.tar.gz in this gitlab script, however in `install-from-dev-pkg.sh` we are defining the name of the tar file as DEV_PKG_SPEC, is this gitlab script wrong or is `install-from-dev-pk`, or am I missing something else here?

Reply:
Yeah I think this is an error. Good catch.